app-commons/src/main/scala/org/bitcoins/commons/jsonmodels/dlc/RoundingIntervals.scala

```diff
@@ -0,0 +1,130 @@
+package org.bitcoins.commons.jsonmodels.dlc
+
+import org.bitcoins.core.currency.Satoshis
+import org.bitcoins.core.util.NumberUtil
+
+import scala.annotation.tailrec
+
+case class RoundingIntervals(intervalStarts: Vector[(BigDecimal, Long)]) {
```

Comment:
nit: Scaladoc please

It would be useful to link to the spec section about rounding too.

What is the `(BigDecimal, Long)` input here?

Reply:
Another high level question: 

What do you we call a `OutcomeValueFunction` (PayoutCurve) with rounding applied? Is there a `RoundedPayoutCurve`?

Reply:
That isn't a thing, the two are separate entities that only come together during processing (a function takes in both an `OutcomeValueFunction` and a `RoundingIntervals`)

Will add a scaladoc though